website/docs/installation/documentdb/rpm.md

```diff
@@ -4,6 +4,80 @@ sidebar_position: 4
+
+We provide different `.rpm` packages for various Red Hat Enterprise Linux (RHEL) and PostgreSQL major versions on [our release page](https://github.com/FerretDB/documentdb/releases/) (e.g. `rhel8-postgresql16-documentdb-<version>-1.el8.x86_64.rpm`)
+
+## Installation
+
+Before installing the DocumentDB extension, make sure to install PostgreSQL and all additional dependencies required by the DocumentDB extension.
+
+With the dependencies installed, you can install the DocumentDB extension using `dnf`.
+
+Download the appropriate DocumentDB `.rpm` package from the release page, then install it by running the following command in your terminal:
+
+```sh
+dnf install -y /path/to/documentdb.rpm
+```
+
+Ensure to replace `/path/to/documentdb.rpm` with the actual path and filename of the downloaded `.rpm` package.
+
+Once installed, update your `postgresql.conf` to load the extension libraries on startup into the default `postgres` database.
+Add the following lines to `postgresql.conf`:
+
+<!-- Keep in sync with https://github.com/FerretDB/documentdb/blob/ferretdb/ferretdb_packaging/10-preload.sh -->
+
+```text
+shared_preload_libraries                      = 'pg_cron,pg_documentdb_core,pg_documentdb'
+cron.database_name                            = 'postgres'
+
+documentdb.enableCompact                      = true
+
+documentdb.enableLetAndCollationForQueryMatch = true
+documentdb.enableNowSystemVariable            = true
+documentdb.enableSortbyIdPushDownToPrimaryKey = true
+
+documentdb.enableSchemaValidation             = true
+documentdb.enableBypassDocumentValidation     = true
+
+documentdb.enableUserCrud                     = true
+documentdb.maxUserLimit                       = 100
+```
+
+Ensure to restart PostgreSQL for the changes to take effect.
+
+Then create the extension by running the following SQL command within the `postgres` database:
+
+```sql
+CREATE EXTENSION documentdb CASCADE;
+```
+
+You can now go ahead and set up FerretDB by following [this installation guide](../ferretdb/rpm.md).
+
+## Updating to a new version
+
+Before [updating to a new FerretDB release](../ferretdb/rpm.md#updating-to-a-new-version), it is critical to install the matching DocumentDB `.rpm` package first.
+
+The following steps are critical to ensuring a successful update.
+
+Download the new `.rpm` package that matches the FerretDB release you are updating to from the release page.
+
+Then install the new package using `dnf`:
+
+```sh
+dnf install -y /path/to/<new-documentdb-package>.rpm
+```
+
+Replace `/path/to/<new-documentdb-package.rpm>` with the actual path and filename of the downloaded `.rpm` package.
+
+After installing the new package, update the DocumentDB extension in your PostgreSQL database.
+To do this, run the following command from within the `postgres` database:
+
+```sql
+ALTER EXTENSION documentdb UPDATE;
```

Comment:
Did this work? Did you test this manually by installing pre-release version, then installing release version or something?

Reply:
So just letting you know that each commit to main branch on documentdb actually builds packages on CI. For example this commit built package before the release https://github.com/FerretDB/documentdb/actions/runs/16871817139/job/47787920106. As you see, the pre-release packages use version number with tilde indicating that it's lower version than the release version such as `0.106.0~ferretdb`. Hence I asked if what's documented in this PR was tested :pray:. Thanks for documenting them though!

<img width="1335" height="511" alt="image" src="https://github.com/user-attachments/assets/21d8211b-5d9d-4a54-aa49-606cb703fc47" />

<img width="1335" height="915" alt="image" src="https://github.com/user-attachments/assets/39e7b50a-e15e-42c8-95fb-5fa723a8a969" />

Reply:
Oh, I didnt know that - thanks! Just tested it with the prebuilt and released version; updates should work normally (although it's still the same version, so nothing was updated.)

```
$ /usr/pgsql-17/bin/psql -d postgres -c 'ALTER EXTENSION documentdb UPDATE;'
NOTICE:  version "0.106-0" of extension "documentdb" is already installed
ALTER EXTENSION
```

Reply:
Yeah you are right. The pre-release version number (ie `0.106.0~ferretdb` built on `ferretdb` branch) is higher than actually higher than release number `0.106.0~ferretdb~2.5.0`. Sorry about that, but thanks for testing. Because tilde `~` makes it lower version https://fedoraproject.org/wiki/PackagingDrafts/TildeVersioning#Basic_versioning_rules.